libs/designsystem/src/lib/components/toast/services/toast.helper.spec.ts

```diff
@@ -79,10 +79,24 @@ describe('ToastHelper', () => {
           await TestHelper.whenReady(ionToast);
           const toastWrapper = ionToast.shadowRoot.querySelector('.toast-wrapper');
           expect(toastWrapper).toHaveComputedStyle({
-            'background-color': getColor(notificationColor),
+            'background-color':
+              messageType !== 'danger' ? getColor(notificationColor) : getColor('warning'),
```

Comment:
Consider readability:
```suggestion
            const expectedColor = messageType === 'danger' ? getColor('warning'): getColor(notificationColor);
            'background-color': expectedColor,
```